src/components/Datagrid/Datagrid.tsx

```diff
@@ -530,16 +530,16 @@ export const Datagrid: React.FC<DatagridProps> = ({
           aria-hidden={true}
           onWheelCapture={hideLinkAndShowAfterDelay}
           onClick={e => {
```

Comment:
move this handler to the separate function and a write test where you pass event and you make assertion what url is being passed to the "navigate" + add comment in the test that navitate uses browser history which handles /dashbaord/ vs /